arista-dcs-7124S/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Arista Switch",
+    "rules": [
+        {
+            "path": "version.Hardwaremodel",
+            "equals": "AristaDCS-7124S-F"
+        },
+        {
+            "path": "version.Internalbuildversion",
+            "equals": "4.13.7M-1877079.4137M.1"
```

Comment:
By the way, this particular Arista switch is EOL but was the one I had available to test on.  Will need to procure other flavors of switches to test out the newer models.